src/snowflake/connector/connection.py

```diff
@@ -276,7 +277,11 @@ class SnowflakeConnection:
 
     OCSP_ENV_LOCK = Lock()
 
-    def __init__(self, **kwargs) -> None:
+    def __init__(
```

Comment:
As we previously discussed, we should also provide an argument for users that they can use a non default path? Maybe not in the first version, but just to make sure you're aware of it.

Reply:
I forgot about this, let me add the capability and add it to `snowflake.connector.connect`